cmd/registry/breaking-change-detector/breaking_change_detector.go

```diff
@@ -0,0 +1,212 @@
+package breakingchangedetector
+
+import (
+	"regexp"
+
+	"github.com/apigee/registry/rpc"
+)
+
+type detectionPattern struct {
+	detectionWeight    int
```

Comment:
Do you have plans for this field in a follow up PR? If not I think we should leave it out.

Reply:
Removed